spec/models/attestation_template_spec.rb

```diff
@@ -87,6 +87,25 @@
     end
   end
 
+  describe 'invalidate attestation if images attachments are not valid' do
+    before do
+      @logo = Rack::Test::UploadedFile.new('spec/fixtures/files/french-flag.gif', 'image/gif')
+      @signature = Rack::Test::UploadedFile.new('spec/fixtures/files/beta-gouv.gif', 'image/gif')
+    end
+
+    after do
+      subject.destroy
+    end
+
+    let(:attestation_template) { AttestationTemplate.create(attributes) }
+    subject { attestation_template.dup }
+
+    context 'with an attestation which has gif files' do
+      let(:attributes) { { title: 't', body: 'b', footer: 'f', activated: true, logo: @logo, signature: @signature } }
+      it { should_not be_valid }
```

Comment:
La syntaxe `should` est un peu ancienne, et plus trop utilisée.

```suggestion
      it { is_expected.to be_valid }
```